proposals/predict_ml_model.json

```diff
@@ -1,7 +1,7 @@
 {
-    "id": "predict_random_forest",
-    "summary": "Predict values based on a Random Forest model",
-    "description": "Applies a Random Forest machine learning model to an array and predict a value for it.",
+    "id": "predict_ml_model",
+    "summary": "Predict values values using a ML model",
```

Comment:
```suggestion
    "summary": "Predict classification or regression values using a ML model",
```

Reply:
Are there any other options that could occur in the future? (Also the summary is too long, we aim for 60 chars in the test suite.)

Reply:
I'd think anything outside of classification/regression is too exotic to be worried about now.

shorter text (if dropping the main verb is ok): "Classification or regression using a ML model"